README.md

```diff
@@ -42,6 +42,23 @@ node = MongoDataNode(group='monolith')
 node.autorun()
 ```
 
+## Examples:
+
+### Base Server
+``` python
+from tyr.servers.server import Server
```

Comment:
I'd recommend against advising users to do

```python
Server(...).autorun()
```

After the server finishes spinning up, there's no way to interact with it. Or if there's an issue, there's no way to rectify it. Or if the user wants to change a value, they will be unable to do that. I'd recommend doing something like

```python
s = Server(...)
s.autorun()
```

Then if it fails to save Chef attributes or the runlist and the problem is your network connection, you can just do

```python
s.bake()
```

If you'd like to get the server's hostname, you can just do

```python
s.hostname
```

If you'd like to check if `chef-client` has finished running, you can do

```python
s.baked()
```

If you'd like to run a command on the server, you can do

```python
s.run('sudo service mongod stop')
```

My point is that by storing the instance in a variable, you can continue to interact with it after running `autorun()`.